### DISCUSSION TITLE
Building the ai_sail code for the AI deck

### DISCUSSION THREAD

**Discussion by shreeram-murali** [bitcraze/discussions#265] [Q&A] [answered]
Hello,

I’m trying to build the codes from this repository [jeguzzi/ai_sail](https://github.com/jeguzzi/ai_sail) to try and stream images from the AI deck at a higher rate. But I’m not really sure how I should start — so I’d appreciate some hints on how to accomplish this. My primary objective is to stream images at a higher rate (at least 30 fps).

Thanks.

**Comment by knmcguire** [comment, marked as answer]
Hi! We had an similar discussion here: https://github.com/orgs/bitcraze/discussions/127

And last blogpost talked about it too: https://www.bitcraze.io/2022/09/using-the-ai-deck-for-crtp-over-wifi-via-cpx/

The short answer is, technically it seems to be possible but the current drivers or implementation it is still limiting it to 7 - 12 fps. More investigation is required to get this higher but due to other projects we will be delayed for the next couple of weeks to really investigate it.

**Reply by krichardsson** [reply]
@shreeram-murali If you think the answer is the proper solution, please consider clicking "Mark as answer". Thanks!

**Reply by shreeram-murali** [reply]
Done! Thank you.
